src/types/facilityData.ts

```diff
@@ -25,12 +25,63 @@ export type Geojson = {
   id: number;
 };
 
-export type Volume = {
+export type AirspaceVolume = {
   labelLocation: [number, number];
   ownership: Ownership;
   geojson: Geojson;
 };
 
-export type Sectors = {
+export type SectorNameMap = {
   [sectorId: string]: string;
 };
+
+// the shape of `~allFacilities.json`
+export interface FacilityCollection {
+  timestamp: number,
+  facilities: FacilityImproved[]
+}
+
+// the shape of the entire group data set
+export interface GroupCollection {
+  timestamp: number,
+  groups: Group[]
+}
+
+// a single group of sectors
+export interface Group {
+  id: number,
+  fir: string,
+  name: string,
+  frequency: string,
+  notes: string,
+  color: string,
+  sectorList: string[]
+}
+
+// TODO: Once everything is working, just use this format, and reformat the data files to match it.
+// facility shape used in comprehensive facility file
+export interface FacilityImproved {
+  firDetails: {
+    id: number, // TODO: Do we need this id field? Presumably not? Why did I put it there?
+    lid: string,
```

Comment:
Not sure what `lid` represents?

Reply:
Thought I had a comment there... It's an established and specific term, actually. Though not particularly widely known. Is an FAA term, Location IDentifier. Each FAA facility has one. Washington ARTCC's lid is `ZDC`.

Currently you are calling ZDC the `.name`, which is rather misleading.

Reply:
Ah. As long as its well documented its fine, however I might even encourage a more verbose name for clarity's sake? Especially since capitalization is inconsistent among the schemas/type files/trpc representation, etc...